todo_app/tests/test_minimal.py

```diff
@@ -0,0 +1,2 @@
+def test_minimal():
+    assert True
```

Comment:
Now that you have your view model tests working we can remove this one

Reply:
removed, thanks.